op-node/rollup/driver/driver.go

```diff
@@ -286,6 +292,21 @@ func (s *Driver) eventLoop() {
 		altSyncTicker.Reset(syncCheckInterval)
 	}
 
+	// upstreamSyncTickerC drives the upstreamSyncTicker, which periodically reconciles
+	// the state against upstream sources when derivation is disabled (unsafeOnly).
+	//
+	// In this mode, the node does not derive from L1; instead, it uses L1 as a mandatory
+	// upstream anchor for its unsafe head, and may optionally import safe/finalized state
```

Comment:
There is a lot of infrastructure value to not having these lite nodes use L1 connections at all. I need to read more closely to understand, but this comment leads me to think the L1 connection remains required?

I suggest we don't need a mandatory anchor from the L1, and can instead use the Safe Source for the L1 as well.